src/main/java/org/jabref/logic/util/io/FileNameUniqueness.java

```diff
@@ -0,0 +1,49 @@
+package org.jabref.logic.util.io;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Optional;
+
+public class FileNameUniqueness {
+
+    /**
+     * Returns an absolute path to a file which does not match with any existing file names
```

Comment:
The description seems to be off as no absolute path is returned but only the file name, right?